src/coreclr/jit/gentree.cpp

```diff
@@ -27353,20 +27220,18 @@ GenTree* Compiler::gtNewSimdTernaryLogicNode(var_types   type,
 //  Arguments:
 //    type                - The return type of SIMD node being created.
 //    op1                 - The SIMD operand.
-//    simdBaseJitType     - The base JIT type of SIMD type of the intrinsic.
```

Comment:
nit: Another place we need to keep the argument comment and just fix it to track the new name